### PR TITLE
UIOR-851 Display invoice line amount and comment on POL in related invoice table

### PR DESCRIPTION
**Pull request by NikitaSedyx**
## Purpose

https://issues.folio.org/browse/UIOR-851

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/POLine/RelatedInvoiceLines/RelatedInvoiceLines.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            visibleColumns={visibleColumns}
+          />
+        )
+      }
+    </Accordion>
+  );
+};
+
+RelatedInvoiceLines.propTypes = {
+  // eslint-disable-next-line react/no-unused-prop-types
```
Comment (AndreiShumski2) [inline src/components/POLine/RelatedInvoiceLines/RelatedInvoiceLines.js:85]: could be removed

File: src/common/constants/api.js [hunk 2]
```diff
@@ -4,3 +4,6 @@ export const INSTANCE_STATUSES_API = 'instance-statuses';
 export const INSTANCE_TYPES_API = 'instance-types';
 export const REASONS_FOR_CLOSURE_API = 'orders/configuration/reasons-for-closure';
 export const UPDATE_ENCUMBRANCES_API = `${ORDERS_API}/:{id}/re-encumber`;
+
+export const INVOICES_API = 'invoice/invoices';
```
Comment (AndreiShumski2) [inline src/common/constants/api.js:8]: We have this in stripes-acq, don't we?

File: src/components/POLine/RelatedInvoiceLines/useLinkedInstances.test.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    useOkapiKy
+      .mockClear()
+      .mockReturnValue({
+        get: (path) => ({
+          json: () => kyResponseMap[path],
+        }),
+      });
+  });
+
+  it('should fetch package POL linked instances', async () => {
```
Comment (AndreiShumski2) [inline src/components/POLine/RelatedInvoiceLines/useLinkedInstances.test.js:63]: linked invoice lines)

File: src/components/POLine/RelatedInvoiceLines/RelatedInvoiceLines.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  amount: <FormattedMessage id="ui-orders.relatedInvoiceLines.amount" />,
+  comment: <FormattedMessage id="ui-orders.relatedInvoiceLines.comment" />,
+};
+const resultFormatter = {
+  invoiceLine: invoiceLine => (
+    <Link
+      data-test-link-to-invoice
+      to={`/invoice/view/${invoiceLine.invoice?.id}/line/${invoiceLine.id}/view`}
+    >
+      {`${invoiceLine.invoice?.folioInvoiceNo}-${invoiceLine.invoiceLineNumber}`}
```
Comment (AndreiShumski2) [inline src/components/POLine/RelatedInvoiceLines/RelatedInvoiceLines.js:39]: `undefined-1` and bad link to invoice line if something goes wrong with the invoice fetching

File: src/common/constants/api.js (hunk 2 above)
Comment (NikitaSedyx) [reply src/common/constants/api.js:8]: no

File: src/components/POLine/RelatedInvoiceLines/RelatedInvoiceLines.js (hunk 4 above)
Comment (NikitaSedyx) [reply src/components/POLine/RelatedInvoiceLines/RelatedInvoiceLines.js:39]: something is wrong that invoice line is fetched for removed invoice :) so it shouldn't be an issue

**Review by AndreiShumski2 — APPROVED** [review]

**Review by usavkov-epam — APPROVED** [review]

**Pull request merged**